### PR TITLE
Added Source Scraper for US/NV Nye County

### PR DESCRIPTION
**Pull request by appastair**
##  Summary

While working on #861, I found a [data source](https://www.nyecounty.net/1066/Coronavirus-COVID-19-Information) for another country so decided to add support for it. :grimacing:

### CODE REVIEW

File: src/shared/scrapers/US/NV/nye-county.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+      const cities = [];
+
+      $(rows).each(function(i, row) {
+        if (
+          $('td', row)
+            .eq(0)
+            .text() === 'Deaths'
+        ) {
+          counties[0].deaths =
```
Comment (jzohrab) [inline src/shared/scrapers/US/NV/nye-county.js:59]: I can't follow what's going on with the next lines ... you're doing
`typeof $('td', row).eq(2).text() === 'number' ? parse.number(typeof $('td', row).eq(2).text())`

that second `typeof` will be a problem. :-)

A temp var would help readability:

```
const d = $('td', row).eq(2).text();
counties[0].deaths = (typeof(d) === 'number') ? parse.number(d) : undefined;
```

File: src/shared/scrapers/US/NV/nye-county.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+          county: this.county,
+          cases: 0,
+          tested: typeof approximateTests === 'number' ? approximateTests : undefined,
+          deaths: 0,
+          recovered: 0,
+          date
+        }
+      ];
+
+      const cities = [];
```
Comment (jzohrab) [inline src/shared/scrapers/US/NV/nye-county.js:51]: Currently the code only returns the counties:

```
[
  {
    county: 'fips:32023',
    cases: 39,
    tested: 1300,
    deaths: undefined,
    recovered: 18,
    date: '2020-05-05'
  }
]
```

Should it return cities?  I actually don't know if we're processing data at that level.

File: src/shared/scrapers/US/NV/nye-county.js (hunk 1 above)
Comment (appastair) [reply src/shared/scrapers/US/NV/nye-county.js:59]: Yeah, this is ghastly. I swear I didn't format it like that. Linter? :speak_no_evil:

Comment (appastair) [reply src/shared/scrapers/US/NV/nye-county.js]: Far tidier than bofore, thanks!

File: src/shared/scrapers/US/NV/nye-county.js (hunk 2 above)
Comment (appastair) [reply src/shared/scrapers/US/NV/nye-county.js:52]: I'm not sure how to return cities.

Comment (jzohrab) [reply src/shared/scrapers/US/NV/nye-county.js:52]: Me neither! :-)

**Comment by jzohrab** [comment]
Thanks very much!  I'll make some minor changes in a new branch based off of this one, and will push it soon.  Cheers @appastair, jz

**Comment by jzohrab** [comment]
Made minor tweaks and merged https://github.com/covidatlas/coronadatascraper/pull/979.

Thank you very much!

**Pull request closed without merging**